packages/main/src/Dialog.ts

```diff
@@ -229,6 +231,8 @@ class Dialog extends Popup {
 	/**
 	 * Defines the header HTML Element.
 	 * <br><br>
+	 * When ui5-bar is used in the Header additional styling is needed. The default Dialog's paddings should be removed.
```

Comment:
Respectively here:
* When a <code>ui5-bar</code> is used in the header, you should remove the default dialog's paddings.